src/variable.cpp

```diff
@@ -143,24 +143,25 @@ Variable::INTERFACE_TYPES Variable::getInterfaceType() const
 
 std::string interfaceTypeToString(Variable::INTERFACE_TYPES interfaceType)
 {
-    std::string str;
+    std::string str = "";
     switch (interfaceType) {
-    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_NONE: {
-        /* Should not ask for the string version of this.
-        With the current codebase there is no way to trigger this case. */
-    }
-    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_PRIVATE: {
+    case Variable::INTERFACE_TYPE_PRIVATE: {
         str = "private";
         break;
     }
-    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_PUBLIC: {
+    case Variable::INTERFACE_TYPE_PUBLIC: {
         str = "public";
         break;
     }
-    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_PUBLIC_AND_PRIVATE: {
+    case Variable::INTERFACE_TYPE_PUBLIC_AND_PRIVATE: {
         str = "public_and_private";
         break;
     }
+    case Variable::INTERFACE_TYPE_NONE: {
```

Comment:
From your reordering in ```prefixToString()```, I thought you wanted to have everything alphabetically ordered? If so, ```Variable::INTERFACE_TYPE_NONE``` ought to be first.